plugins/fastboot/fu-fastboot-device.c

```diff
@@ -19,11 +19,13 @@
 #define FASTBOOT_EP_IN			   0x81
 #define FASTBOOT_EP_OUT			   0x01
 #define FASTBOOT_CMD_BUFSZ		   64 /* bytes */
+#define FASTBOOT_US_TO_MS		   1000
```

Comment:
any reason can't use the glib macros?  I guess they go to seconds, do you really need ms?

Reply:
GLib macros go from microseconds to seconds only...
Milliseconds seems like a good compromise regarding precision and readability.

Reply:
Got it, thanks.